GoogleForCreators/web-stories-wp
# Design Panel: Bugfix - reset stored height of saved colors panel

## Context

>The issue is, that before the 1.5 release (so in 1.4), the saved color panel height was only a single line of color circles, so it was much smaller. And yes, that height was saved in local settings. When a user upgrades, the saved height persists, ignoring that the panel now should never be this tiny.

https://xwp.slack.com/archives/CCCAUAH6F/p1616176802029600?thread_ts=1616176594.029300&cid=CCCAUAH6F

## Summary

Changes the `presetType` for the colors panel. This changes the local storage name and resets the saved sizing values.

## Relevant Technical Choices

n/a

## To-do

none

## User-facing changes

Opening the colors panel will ignore the current saved sizing data. This can then be overwritten.

1. In this gif I set the height in this panel. This sets the value in local storage
2. I reload the page, but the presetType has been changed in my local code
3. You see that the height does not persist
4. Changing the height and reloading again persists the new height (since the data has a different key in local storage)
<img src="https://user-images.githubusercontent.com/22185279/111828131-73010380-88b0-11eb-997a-c7426a7558ee.gif" width="500px" />


## Testing Instructions

1. Open the editor
2. Click on the background or an element with color
3. Open the dev tools. There should now be two keys in local storage for colors:
    - `web_stories_ui_panel_settings:stylepreset-colors` (new)
    - `web_stories_ui_panel_settings:stylepreset-color` (old)
4. Expanding the panel should update the new key's value

### QA




This PR can be tested by following these steps:

1. Open the editor
2. Click on the background or an element with color
3. Open the dev tools. There should now be two keys in local storage for colors:
    - `web_stories_ui_panel_settings:stylepreset-colors` (new)
    - `web_stories_ui_panel_settings:stylepreset-color` (old)
4. Expanding the panel should update the new key's value
**note** if you don't see the `Saved Colors` panel, try clicking on a different element or adding a new text element to get the panel to show up in the design tab.

### UAT






## Reviews

### Does this PR have a security-related impact?

no

### Does this PR change what data or activity we track or use?

This pr changes the way we store the sizing data for the saved colors panel.

### Does this PR have a legal-related impact?

no

## Checklist




---



Fixes #6740
